hrjobcontract/xml/organisation_contact_install.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<CustomData>
+
```

Comment:
We probably don't need this extra space. Maybe an extra space between the option group items would be good.